protos/log_files/log_files.proto

```diff
@@ -34,6 +42,14 @@ message DownloadLogFileResponse {
     ProgressData progress = 2; // Progress if result is progress
 }
 
+message DownloadLogFileRequest {
+    Entry entry = 1; // Entry of the log file to download.
+    string path = 2; // Path of where to download log file to.
+}
+
+message EraseLogFilesRequest {}
```

Comment:
Just a small question here: what about `EraseAllLogFilesRequest`? I have read this file multiple times and often I would think "oh wait, it's missing the file argument", before I realised it was for all of them... But maybe that's just me :laughing:

Reply:
Based that name on the according mavlink message `LOG_ERASE`, but you have a valid point!